packages/runtime/src/runtime.ts

```diff
@@ -66,6 +69,38 @@ export class Runtime {
     this.timestamp = 1;
   }
 
+  get defaultBalance (): number {
+    return 1e9; // 1000 Algos
```

Comment:
Let's use 100 ALGO's (`100 * 1e6`) as the default balance. (same as the test accounts on a ganache mainnet fork)